### ISSUE TITLE
Adding a texture file in a fresh project from "Level Settings/Texture file"s doesn't make it available right away in the texture panel

### ISSUE THREAD

**Issue by diebasis72** [MontyTRC89/Tomb-Editor#931]
When opening a fresh Tomb Editor and add a texture file from Level Settings > Texture Files, it won't be available right away, you have to select it first in the texture panel.

Would be nice if it were in the panel right away.

![Unbenannt 2](https://github.com/user-attachments/assets/b2bf559f-5f6d-4970-81d8-f16a77c2180f)

**Comment by Raildex** [comment]
I do not understand the problem?
Level Settings need to be applied first before the changes take place

**Comment by diebasis72** [comment]
> I do not understand the problem? Level Settings need to be applied first before the changes take place

I did hit "Apply". But you have to select it from panel anyway afterwards. For better understanding, I aligned the two screenshots next to each other. Sorry, if it may have looked confusing.

**Comment by diebasis72** [comment]
> ah, so you want the texture panel to be populated after you hit apply, so the texture is instantly visible?

Exactly. Recently I added a texture set this way and thought I didn't do something wrong because it didn't show up. So I added it a second time only to find out later that it was already in the Texture panel but I I didn't select it to show up in the Texture panel. 

So I thought maybe this may be confusing for other builders too.
